mozilla_version/gecko.py

```
@@ -487,7 +487,7 @@ def _create_bump_kwargs(self, field):  # noqa C901
                 raise ValueError(
                     "Cannot bump the major number past last known major ESR. We don't "
                     "know it yet."
-                )
+                ) from None
```

Huh, I didn't know you could raise an exception from None